modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/CassandraAuthorizationRepository.java

```diff
@@ -387,4 +390,76 @@ public UserPermissions checkSuperAdminPermissions(UserInfo.Username userID, Appl
         }
         return resultList;
     }
+
+    @Override
+    public void assignUserToSuperAdminRole(UserInfo candidateUser) {
+
+        LOGGER.debug("Adding user {} as superadmin", candidateUser);
+
+        String superAdminRole = Role.SUPERADMIN.toString().toLowerCase();
+        String userId = candidateUser.getUsername().toString();
```

Comment:
@abhishekjain88 I see we are using userID more often in the code. It will be more consistent to change userId to userID. just a suggestion.